src/test/java/gov/healthit/chpl/aqa/pageObjects/ChplAPIPage.java

```diff
@@ -0,0 +1,54 @@
+package gov.healthit.chpl.aqa.pageObjects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+/**
+ * Class ChplAPIPage definition.
+ */
+public final class ChplAPIPage {
+    private ChplAPIPage() {}
+    private static WebElement element = null;
+
+    /**
+     * Returns element holding main content.
+     * @param driver WebDriver
+     * @return element holding main content
+     */
+    public static WebElement mainContent(final WebDriver driver) {
```

Comment:
I wonder if we could do something where we have a "BasePage" class that has this element in it, along with the navigation ones, and have all of our specific pages extend from that. Then all pages would have a reference to the main-content as well as navigation things, and we'd have the code in only one place